rest-server/src/rest_server.cpp

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018   Vaniya Agrawal, Ross Arcemont, Kristofer Hoadley,
+//                  Shawn Hulce, Michael McCulley
+#include <iostream>
+#include <string>
+#include "another_controller.hpp"
+#include "example_controller.hpp"
+#include "user_controller.hpp"
+#include "data_access_interface.hpp"
+#include "data_access_object.hpp"
+using aoi_rest::DataAccessInterface;
+using aoi_rest::DataAccessObject;
+using aoi_rest::AnotherController;
+using aoi_rest::ExampleController;
+using aoi_rest::UserController;
+
+int main(int argc, const char * argv[]) {
+    std::cout << "Starting aoi rest server" << std::endl;
+    DataAccessInterface* user_dao = &DataAccessObject::Instance();
```

Comment:
Does this need to use a pointer?

Reply:
I believe it does, since it is a single object, any access to it would be pointers no?